docs/resources/User.md

```diff
@@ -1,14 +1,14 @@
 # Users Resource
 
 Users in Discord are generally considered the base entity. Users can spawn across the entire platform, be members of
-guilds, participate in text and voice chat, and much more. Users are separated by a distinction of "bot" vs "normal." Although they are similar, bot users are automated users that are "owned" by another user. Unlike normal users, bot users do
+guilds, participate in text and voice chat, and much more. Users are separated by a distinction of "bot" vs. "normal." Although they are similar, bot users are automated users that are "owned" by another user. Unlike normal users, bot users do
 _not_ have a limitation on the number of Guilds they can be a part of.
 
 ## Usernames and Nicknames
 
 Discord enforces the following restrictions for usernames and nicknames:
 
-1.  Names can contain most valid unicode characters. We limit some zero-width and non-rendering characters.
+1.  Names can contain the most valid Unicode characters. We limit some zero-width and non-rendering characters.
```

Comment:
no, the original is correct. this is talking about "most unicode characters that are valid", not "unicode characters that are the most valid"